Sources/Testing/Running/Configuration.swift

```diff
@@ -128,17 +128,38 @@ public struct Configuration: Sendable {
   ///
   /// By default, all tests are run and no filter is set.
   public mutating func setTestFilter(toMatch selection: Set<Test.ID>?) {
```

Comment:
After this change, is there any code coverage of this overload?

Reply:
This method has one callsite, in `XCTestScaffold.runAllTests(hostedBy:)`. I chose not to modify this method since it's public API, and we don't want to expose the concept of hidden tests to clients since that's internal to the project.

Reply:
Done.